sources/packages/web/src/composables/useSnackBar.ts

```diff
@@ -0,0 +1,48 @@
+import { SnackBarType } from "@/types";
+import useEmitter from "./useEmitter";
+
+export function useSnackBar() {
+  const emitter = useEmitter();
+
+  const success = (content: string, displayTime = 5000): void => {
+    const snackbarOptions = {
+      show: true,
+      type: SnackBarType.success,
+      content: content,
+      displayTime: displayTime,
+    };
+    emitter.emit("snackBar", snackbarOptions);
+  };
+
+  const error = (content: string, displayTime = 5000): void => {
+    const snackbarOptions = {
+      show: true,
+      type: SnackBarType.error,
+      content: content,
+      displayTime: displayTime,
+    };
+    emitter.emit("snackBar", snackbarOptions);
```

Comment:
This `emitter.emit("snackBar", snackbarOptions);` can be centralized in a simple method call to avoid repeating the code (and the string) for every method.